vllm/model_executor/layers/quantization/compressed_tensors/utils.py

```diff
@@ -80,7 +80,10 @@ class QuantizationArgs(BaseModel):
     )
 
 
-def is_activation_quantization_format(format: str) -> bool:
+def is_activation_quantization_format(format: Union[str, None]) -> bool:
+    if format is None:
+        return False
```

Comment:
Why is this needed?

Reply:
Thought to make it explicit, but since `None` is not the in the list below we can also get rid of it. Removed in the last force push